docs/tutorials/tutorial_experience.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<p>I think in the beginning it would be nice to underline the connection between the initial conditions tutorials a bit more to emphasize that these are similar 'type' of parameters in the <code>params</code> DataFrame in regards to how they affect the model and data. You can also link to the <strong>observables</strong> tutorial here.</p><p> </p><p>Furthermore, for this tutorial I feel like we could breach the Robinson world for a second in the introduction to discuss the motivation for including initial experience a bit more (i.e. for analyses of human capital accumulation and occupational choice, heterogeneity in initial experience is something we often observe and want to control for). If you can think of an example that fits Robinson's story, even better, but I can't think of one currently. Otherwise I feel like including this specific source of heterogeneity might seem a bit random maybe/ not as obvious as observables? What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I would also prefer the tutorials not to really "build on each other", just like @tobiasraabe  wrote below --- apart from the very first one where the basic model is introduced. So maybe an alternative could be to write the same introduction on initial conditions in both the observables and experience tutorials (and lagged choices), _but_ enrich these introductions with some motivation which goes beyond the basic Robinson Crusoe economy, just like you wrote.

Reply:
Yes I totally agree with you guys that the tutorials should be self contained and not presented in a particular order. I mainly meant that it's nice if the connection between these different parameters becomes clear,  mentioning it in the beginning like you suggest is enough. I think linking to the other two in each one would be nice as well but we can leave that out of course if you think it's too much :).